src/views/OverviewView.js

```diff
@@ -1,25 +1,53 @@
-import { OrthographicView } from '@deck.gl/core';
+/* eslint-disable max-classes-per-file */
+import { Controller, OrthographicView } from '@deck.gl/core';
 import VivView from './VivView';
 import { OverviewLayer } from '../layers';
 import { makeBoundingBox, getVivId } from './utils';
 
+class OverviewState {
+}
+
+class OverviewController extends Controller {
+  constructor(props) {
+    super(OverviewState, props);
+    this.events = ["click"];
+  }
+
+  handleEvent(event) {
+    if (event.type !== "click") {
+      return;
+    }
+    let [x, y] = this.getCenter(event);
+    const { width, height, zoom, scale } = this.controllerStateProps;
+    if (x < 0 || y < 0 || x > width || y > height) {
+      return;
+    }
+    const scaleFactor = 1 / (2 ** zoom * scale);
+    x *= scaleFactor;
+    y *= scaleFactor;
+    if (this.onViewStateChange) {
+      this.onViewStateChange({ viewState: { Overview: { target: [x, y, 0] } } });
+    }
```

Comment:
```suggestion
    const { id } = this;
    if (this.onViewStateChange) {
      this.onViewStateChange({
        viewId: id,
        viewState: { id, target: [x, y, 0] }
      });
    }
```

I think this is what we want, modulo prettier?  Did what you have work with Avivator?  It might have been relying on some of the old overriding code you had.

Reply:
I'm not sure if that would work because `viewState.target` has a meaning for `OverviewView`. I'll test it in my own implementation. However, I just noticed a mistake that I made in string replace, instead of `Overview`, that should have been `navigation`.

Reply:
It works without `navigation` so that I'll remove that.

Why do you want to add `id` and `viewId`? deck.gl's `ViewManager` automatically adds `viewId`: https://github.com/visgl/deck.gl/blob/master/modules/core/src/lib/view-manager.js#L240
`VivViewer._onViewStateChange` adds that to `viewState` as `id`.

Reply:
@andreasg123 I didn't know that about `id` so you could remove that.  I think the `id` should be `overview` not `navigation` since that is the internally used `id`.  Your other change wrt. `navigation` had `viewState.navigation` but it seems like you're saying we should make `viewState.id` to be `navigation` which is different.